docs/customization/model.rst

```diff
@@ -377,10 +409,12 @@ in our case it will be the ``SyliusShippingBundle`` -> ``sylius_shipping``.
         resources:
             shipping_method:
                 classes:
-                    model: App\Entity\ShippingMethod
+                    model: App\Entity\Shipping\ShippingMethod
                 translation:
                     classes:
-                        model: App\Entity\ShippingMethodTranslation
+                        model: App\Entity\Shipping\ShippingMethodTranslation
```

Comment:
We are missing the interface configs here and above I suppose?

Reply:
And here